posts/infinity-cosmos-announcement.md

```diff
@@ -0,0 +1,38 @@
+
+We propose an alternate approach to the formalization of ∞-category theory starting from the axiomatic notion of an [∞-cosmos](https://ncatlab.org/nlab/show/infinity-cosmos). An ∞-cosmos is an axiomatization of the universe in which ∞-categories live as objects. The axioms are somewhat ad-hoc and chosen for largely pragmatic reasons. In particular, the axioms are stated in the language of 1-category theory and simplicially enriched category theory — topics that are well-covered by Mathlib — rather than in the language of ∞-category theory itself. The theory of ∞-cosmoi enables a "model-independent" development of the theory of ∞-categories. On the one hand, various models such as quasi-categories and complete Segal spaces assemble into ∞-cosmoi, so theorems about ∞-categories in an ∞-cosmos apply to the most popular models. At the same time, the ∞-categorical notions developed in an ∞-cosmos are "synthetic" or "formal" as opposed to the original "analytic" definitions, which were typically expressed in the "coordinates" of a particular model.
+
+Much of the development of the theory of ∞-categories takes place not in the full ∞-cosmos but in a quotient 2-category whose objects are ∞-categories, whose morphisms are ∞-functors, and whose 2-cells are ∞-natural transformations. We call this the homotopy 2-category since it is a quotient of the ∞-cosmos (which is an (∞,2)-category of a particular form) much like an (∞,1)-category has a quotient homotopy (1-)category. For instance, the existing Mathlib notions of equivalences and adjunctions in a bicategory specialize to define equivalences and adjunctions between ∞-categories when interpreted in the homotopy 2-category.
+
+# Initial scope
+
+The initial aims of this project are relatively modest, though with sufficient interest its ambitions could expand considerably. In particular, our initial intention is not to attempt the difficult theorem of proving that the cartesian closed category of quasi-categories defines an example of an ∞-cosmos; among other things, the cartesian closure of this subcategory has not yet been proven. Instead, our intention is to create a large "bounty" to reward anyone who succeeds in this task in the form of a large cache of ∞-categorical theorems that will follow immediately.
+
+# Team behind ∞-Cosmos
+
+This project is being co-lead by [Dominic Verity](https://github.com/dom-verity) and [Mario Carneiro](https://github.com/digama0).
```

Comment:
```suggestion
This project is being co-led by [Dominic Verity](https://github.com/dom-verity) and [Mario Carneiro](https://github.com/digama0).
```

But actually this sentence can be misinterpreted as "there are two leaders of the project, Verity and Carneiro, and Riehl is just somehow doing the announcement of the project" and my understanding is that the truth is "there are three leaders", so I would suggest explicitly mentioning Riehl here too.

Reply:
You are right, thanks.
I also wanted to add Emily, but I wasn't sure how to formulate it in proper english...

My understanding was that Emily is the head of this project and Verity and Carneiro are co-leading it.
What about:

This project was started/envisioned/initiated by Emily and is beeing co-led by Verity and Carneiro.

Sorry if it sounds rough, english is not my first language